include/common_robotics_utilities/time_optimal_trajectory_parametrization.hpp

```diff
@@ -121,15 +121,13 @@ class Trajectory
   Trajectory(const std::list<Eigen::VectorXd>& waypoints,
              const Eigen::VectorXd& max_velocity,
              const Eigen::VectorXd& max_acceleration,
-             const double max_deviation,
-             const double timestep);
+             const double max_deviation = 0.0,
+             const double timestep = 0.001);
 
-  Trajectory(const Path &path_,
-             const Eigen::VectorXd& max_velocity_,
-             const Eigen::VectorXd& max_acceleration_,
-             double timestep = 0.001);
-
-  ~Trajectory(void);
+  Trajectory(const Path& path,
+             const Eigen::VectorXd& max_velocity,
+             const Eigen::VectorXd& max_acceleration,
+             const double timestep = 0.001);
```

Comment:
BTW Consider sharing default constants with an explicitly declared constant, e.g. `kDefautlTimestamp`?

Reply:
I will deal with them more broadly in the future #12